src/promptflow/promptflow/_utils/multimedia_utils.py

```diff
@@ -221,14 +225,19 @@ def _process_recursively(value: Any, process_funcs: Dict[type, Callable] = None,
 def load_multimedia_data(inputs: Dict[str, FlowInputDefinition], line_inputs: dict):
     updated_inputs = dict(line_inputs or {})
     for key, value in inputs.items():
-        if value.type == ValueType.IMAGE:
-            if isinstance(updated_inputs[key], list):
-                # For aggregation node, the image input is a list.
-                updated_inputs[key] = [create_image(item) for item in updated_inputs[key]]
-            else:
-                updated_inputs[key] = create_image(updated_inputs[key])
-        elif value.type == ValueType.LIST or value.type == ValueType.OBJECT:
-            updated_inputs[key] = load_multimedia_data_recursively(updated_inputs[key])
+        try:
+            if value.type == ValueType.IMAGE:
+                if isinstance(updated_inputs[key], list):
+                    # For aggregation node, the image input is a list.
+                    updated_inputs[key] = [create_image(item) for item in updated_inputs[key]]
+                else:
+                    updated_inputs[key] = create_image(updated_inputs[key])
+            elif value.type == ValueType.LIST or value.type == ValueType.OBJECT:
+                updated_inputs[key] = load_multimedia_data_recursively(updated_inputs[key])
+        except InvalidImageInput as ex:
```

Comment:
Is it good to catch `InvalidImageInput` only? Should we catch more other exceptions?

Reply:
Updated.